README.md

```diff
@@ -98,3 +99,9 @@ This project has adopted the [im-open's Code of Conduct](https://github.com/im-o
 ## License
 
 Copyright &copy; 2021, Extend Health, LLC. Code released under the [MIT license](LICENSE).
+
+[create-release]: https://github.com/actions/create-release
```

Comment:
I didn't see you referencing this in the README, so it might be safe to remove it.